docs/synthetics/status-pages.md

```diff
@@ -0,0 +1,37 @@
+title: Status Pages
+description: Communicate the status of your services to your customers using status pages
+
+Using Sematext Synthetics you can create public status pages to share the status and response time of your services to your customers.
+
+## Create Status Page
+
+You can create multiple status pages under your account. Navigate to the `Synthetics -> Status Pages` screen to create a status page. 
+
+![Create Status Page](../images/synthetics/status-page-create.png)
+
+The following fields are required to create a status page:
+
+* **Monitors** - List of monitors to be included in the status page. All the active monitors from Apps for which the account has OWNER or ADMIN role can be included in the status page. A monitor can be included in multiple status pages.
+* **Name** - Unique name for the status page.
+* **Header Name** - Company name to be displayed in the status page header.
+* **Header Logo** - URL of the company logo to be displayed in the status page header. An image with a height of 40px and white background is recommended.
```

Comment:
What about image width?  600px is also OK?  What will happen if the image doesn't fit or height-width ratio is not ideal?  White background?  Transparent wouldn't work or be more universal (say if later on we let people select their bg colour)?

Reply:
CC  AFAIK we are limiting the width CC @hamer86 . Yes, the transparent background will work better. I will change it.

Reply:
We don't limit image width. But we do limit image height to 40px in CSS. But the image is limited by the header container. And the image can't be wider. in that case, the header name will go into the next row.